numpy/core/src/multiarray/dragon4.c

```diff
@@ -1060,12 +1175,14 @@ Dragon4(const npy_uint64 mantissa, const npy_int32 exponent,
     DEBUG_ASSERT(bufferSize > 0);
```

Comment:
Did you test that using this assert (line 39) still passes tests?

Reply:
Good idea, I hadn't actually.

I just did now and tests pass. (dragon4 related tests, anyway).